### DISCUSSION TITLE
Crazyflie 1.0 doesn't turn on

### DISCUSSION THREAD

**Discussion by Dracarnion** [bitcraze/discussions#743] [Q&A]
Hi folks - I know it's a blast from the past but it has been serving me well all these years.  I recently traveled with it and perhaps something got messed up.  It no longer turns on or shows any sign of life when the power button is pressed.  If I plug it into a PC, I get lights including 5/6 green flashes followed by a solid green, blue and blinking red.  The props don't spin as they normally would when I plug it in. 

I tried unplugging the battery and plugging it back in while it is connected to the PC.  I attempted to download the cf client on my PC to try to flash the firmware but it failed to install 

 Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [26 lines of output]
      Traceback (most recent call last):
        File "C:\Users\Jesse\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\Jesse\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Jesse\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Jesse\AppData\Local\Temp\pip-build-env-20yyslgh\overlay\Lib\site-packages\setuptools\build_meta.py", line 341, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Jesse\AppData\Local\Temp\pip-build-env-20yyslgh\overlay\Lib\site-packages\setuptools\build_meta.py", line 323, in _get_build_requires
          self.run_setup()
        File "C:\Users\Jesse\AppData\Local\Temp\pip-build-env-20yyslgh\overlay\Lib\site-packages\setuptools\build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 141, in <module>
        File "C:\Users\Jesse\AppData\Local\Temp\pip-install-g_px8268\pyzmq_7a63e1ccbcd54edaaf1231954b80902f\buildutils\misc.py", line 83, in locate_vcredist_dir
          vcvars = msvc.msvc14_get_vc_env(get_platform())
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Jesse\AppData\Local\Temp\pip-build-env-20yyslgh\overlay\Lib\site-packages\setuptools\msvc.py", line 210, in msvc14_get_vc_env
          return _msvc14_get_vc_env(plat_spec)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\Jesse\AppData\Local\Temp\pip-build-env-20yyslgh\overlay\Lib\site-packages\setuptools\msvc.py", line 164, in _msvc14_get_vc_env
          raise distutils.errors.DistutilsPlatformError(
      setuptools._distutils.errors.DistutilsPlatformError: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

Please let me know if you can provide any further troubleshooting steps.  I know I'm long overdue to upgrade to a new drone, but this will always be my first and I'd love to keep it running.

Thanks

edit: Just noticed that if I plug in the battery while it's plugged in to the PC, one of the props rotates once

**Comment by tobbeanton** [comment]
Could it be the battery that is dead?

**Comment by Dracarnion** [comment]
I can try. Will the 250mAh battery be compatible with the crazy flie 1.0?

**Reply by ataffanel** [reply]
Electrically, any single-cell LIPO battery would be compatible. However the Crazyflie 2.x batteries do not have the same connector so you would have to do some soldering.

**Comment by Dracarnion** [comment]
Ok.  So, after closer examination of the battery, some of the wires got
frayed during travel.  I have wrapped them and I have found that the
crazyflie powers up as expected.  However, as soon as the battery is
plugged in, one of the props spins continuously.  The others stay off until
I push the power button and then rotate once as expected.  The other prop
continues to spin continuously.  Do you still suspect an issue with the
battery or could this be a short in the wiring on the crazyflie PCB due to
travel?

**Reply by tobbeanton** [reply]
That sounds like a short or that the mosfet driving the motor has broken but most likely a short.

**Comment by Dracarnion** [comment]
Ok so this thing has been soldered for like 6 years.  I tried resoldering
that prop but now they don't spin at all.  I may have fried the pads but I
proved out that the other ones work with the battery connected so I think
everything else would be fine if not for that motor.  I had an extra motor
so I verified the issue wasn't the motor.  What's my recourse?  Can I get a
replacement crazyflie 1.0 PCB or are they discontinued?  I don't want to
have to purchase a whole new kit since everything else seems to work fine
except for the connections to that one motor.

Thanks for your help!

**Reply by tobbeanton** [reply]
Can you check that you have the battery voltage on one of the motor connectors when the battery is connected?

The CF1.0 is unfortunately discontinued since many years.

**Comment by Dracarnion** [comment]
I do have voltage between the pads for each of the motor connectors with
the battery connected.  However one of the motors continues to be unable to
turn on

**Reply by tobbeanton** [reply]
That indicates that the mosfet is the problem, or potentially the MCU pin but that is less likely. Now you say the one of the motors does not turn on, so the state has changes since one was spinning all the time? Can you perhaps in detail describe the current state, that you know of, and I can advice if it might be possible to save the board?

**Comment by Dracarnion** [comment]
Yup after attempting to resolder the motor that was constantly spinning, it
now doesn't spin at all.

If I connect the battery and turn on the crazy fly, three of the motors
rotate once and the lights flicker as normal when it is turned on, but the
one motor does not move at all.

How can I check the MOSFET? I was concerned I damaged the pads trying to
melt the old solder, but I do have continuity between the pads where that
motor is connected.

Thanks for your help

**Reply by tobbeanton** [reply]
What motor is it? Then I can point out how to short the mosfet so you can test that the motor is spinning.

**Reply by tobbeanton** [reply]
Also I recommend checking the old [forum.bitcraze.io](https://forum.bitcraze.io/search.php?keywords=mosfet). This will have posts about broken mosfets. It might be hard to distinguish CF1.0 from 2.0 topics but the theory is the same.

**Reply by tobbeanton** [reply]
e.g. [this](https://forum.bitcraze.io/viewtopic.php?p=4169&hilit=mosfet#p4169) is a good related topic.
